packages/xsnap/test/test-xsnap.js

```diff
@@ -133,7 +133,7 @@ test('print - start compartment only', async t => {
     const send = it => issueCommand(ArrayBuffer.fromString(it));
     print('print:', 123);
     try {
-      (new Compartment()).evalate('print("456")');
+      (new Compartment()).evaluate('print("456")');
     } catch (_err) {
       send('no print in Compartment');
     }
```

Comment:
Oh! Thanks for catching this!

@kriskowal @dckc Did this succeed before this fix because throwing was considered a success condition, without independent of why it was thrown?

Likewise on line 153 below.

Reply:
Yeah, I think that's right. Oops. We should probably assert that `print` appears in the error text somehow.